.github/workflows/ios.yml

```diff
@@ -14,21 +14,26 @@ jobs:
       contents: write
 
     steps:
-      - uses: actions/setup-go@v1
-        with:
-          go-version: "1.18.3"
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
+      - name: Get GOVERSION content
+        id: goversion
+        run: echo ::set-output name=version::$(cat GOVERSION)
+
+      - uses: actions/setup-go@v1
+        with:
+          go-version: "${{ steps.goversion.outputs.version }}"
+
       - run: |
           echo -n $PSIPHON_CONFIG_KEY > ./internal/engine/psiphon-config.key
           echo $PSIPHON_CONFIG_JSON_AGE_BASE64 | base64 -d > ./internal/engine/psiphon-config.json.age
         env:
           PSIPHON_CONFIG_KEY: ${{ secrets.PSIPHON_CONFIG_KEY }}
           PSIPHON_CONFIG_JSON_AGE_BASE64: ${{ secrets.PSIPHON_CONFIG_JSON_AGE_BASE64 }}
 
-      - run: ./mk XCODE_VERSION=12.4 ./MOBILE/ios
+      - run: ./mk EXPECTED_XCODE_VERSION=12.4 ./MOBILE/ios
```

Comment:
This is wrong

Reply:
This change (from an experimental branch) will be introduced by a subsequent PR.